terraform/README.md

```diff
@@ -186,29 +195,29 @@ Terraform module.
     inputs/parameters to the module definition. See the `variables.tf` file for
     the full list of configuration options.
 
-1.  (Optional if alerting is enabled): Manually create Google Cloud Monitoring
```

Comment:
You can leave these all as "1". Markdown does the right thing.

Reply:
hmm must be my markdown vs plugin